packages/@lwc/engine-server/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@lwc/engine-server",
-    "version": "2.20.0",
+    "version": "2.20.0-canary-ssr.1",
```

Comment:
I assume you're publishing packages to a local npm server for testing or something? This looks like a mistake. 🙂

Reply:
I've been sharing canary builds with LWR as they move forward, tweaking the implementation as needed. There's a TODO in the PR description to drop the commit that releases the canary version. That's being left until later, since I may release another canary before this branch is merged.